sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/UcfgBuilder_Instructions.cs

```diff
@@ -1488,5 +1488,54 @@ public void BuildWebHost2(string[] args)
 }";
             UcfgVerifier.VerifyInstructions(code, "BuildWebHost2");
         }
+
+
+        [TestMethod]
+        public void AddAssignment()
+        {
+            const string code = @"
+namespace Namespace
+{
+    public class Class1
+    {
+        public string fieldString;
+        public string PropertyString { get; set; }
+
+        public void Foo(string parameterString)
+        {
+            var localString = ""foo"";              // localString := __id [ const ]
+
+            localString += parameterString;         // %0 := __concat [ localString parameterString ]
+                                                    // localString := __id [ %0 ]
+
+            localString += ""foo"";                 // %1 := __concat [ localString const ]
```

Comment:
Tests:
* _localString += field, property, method that returns string_
* ?same with field and property on the left-hand side?